services/cinema_hall.py

```diff
@@ -0,0 +1,17 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> CinemaHall:

```

Comment:
The return type annotation for `get_cinema_halls` is incorrect. It should be `-> QuerySet[CinemaHall]` instead of `-> CinemaHall` because `CinemaHall.objects.all()` returns a QuerySet, not a single instance.